src/meetingsession/MeetingSessionConfiguration.ts

```diff
@@ -138,27 +138,35 @@ export default class MeetingSessionConfiguration {
    * ```
    */
   constructor(createMeetingResponse?: any, createAttendeeResponse?: any) { // eslint-disable-line
+    if (createAttendeeResponse) {
```

Comment:
nit: Convert the attendee response within one `if` below, similar to the meeting response.